ui/src/components/LandingPage.css

```diff
@@ -1,5 +1,5 @@
 .title {
-    padding-top: .5em;
-    margin-bottom: .5em;
-    margin-left: 2em;
+    margin-left: 5%;
```

Comment:
@laurafitzgerald is there a reason for the change in units used for the styling?

Reply:
yes, i believe we'll have less inconsistency. When I set both the table and the Header to have 2em they didn't line up. When using percentage they will line up.

Reply:
@damienomurchu does it fix the discrepancy on your end?

Reply:
Understand. Ideally I think we would look at using `rem` (which is used by PF, and the spacing for the components specified usually in `rem` at the bottom of each component page). If we need to get this merged in due to time constraints we can go with `%` but I think `rem` might be better (but we can refactor in future).

Reply:
Left comment above